cli/src/main/kotlin/com/bazel_diff/cli/GenerateHashesCommand.kt

```diff
@@ -64,6 +65,14 @@ class GenerateHashesCommand : Callable<Int> {
     )
     var bazelCommandOptions: List<String> = emptyList()
 
+    @CommandLine.Option(
+        names = ["--fineGrainedHashExternalRepos"],
```

Comment:
```suggestion
        names = ["--fine-grained-hash-external-repos"],
```

Reply:
Thanks! Updated and also added an integration test covering this flag.